src/main/resources/META-INF/rust-core.xml

```diff
@@ -983,6 +983,11 @@
                   topic="org.rust.cargo.project.model.CargoProjectsService$CargoProjectsListener" />
     </projectListeners>
 
+    <applicationListeners>
```

Comment:
nit: I'd personally move `applicationListeners` before `projectListeners` because they have a "wider" scope. But it's up to you